### PR TITLE
Update doc to make clear where the gathered information comes from

### PR DESCRIPTION
**Pull request by JuliaBrigitte**
Update doc to make clear the information displayed is only as up to date as the last run for AWS Trusted Advisor. Also put links in README to other doc available in repo, because that would have made my life easier. Some info about how to debug added too.

### CODE REVIEW

File: script/debug [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (akash1810) [inline script/debug:1]: In the README, we've said:

```bash
./script/start --debug
```

Which implies this extra debug script isn't needed?

File: README.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ presents the data collected by those processes.
 It also provides an interface on some markers of a watched AWS
 account's health from a security point of view.
 
+## Trusted Advisor
+Security HQ uses information from AWS Trusted Advisor.
+This might not be as up to date as one might wish and because noticeable for S3 buckets.
```
Comment (kenoir) [inline README.md:16]: Can we clarify what the update cadence is, whether it's possible to refresh this data and what the restrictions are around that?

Comment (kenoir) [inline README.md:16]:
```suggestion
This might not be as up to date as one might wish and may be noticeable for S3 buckets.
```

Comment (JuliaBrigitte) [reply README.md]: I put the comment more it to give further onlookers a hint, not because I yet know how it works.

**Review by akash1810 — APPROVED** [review]
Not a blocker, can be done in a future PR: I wonder if the AWS SDK returns a last updated timestamp that we can show?

**Pull request merged**